customizations/engage/templates/frame.haml

```diff
@@ -266,7 +266,7 @@
     #header
       &nbsp;
 
-    %a#big-logo.logo{href:"/", title:"{{brand.name}}"}
+    %a#big-logo.logo{href:"{% if brand.logo_link %}{{brand.logo_link}}{%else%}{%endif%}", title:"{{brand.name}}"}
```

Comment:
How does this jive with https://github.com/istresearch/rapidpro/blob/develop/templates/frame.haml#L255?

Reply:
@andrewkcarter I removed the subdir code because adding an elif request.subdir... actually complicates configuration since the default env value for  BRANDING_LOGO_LINK is a "/" the elif wouldn't execute:

`{href:"{% if brand.logo_link %}{{brand.logo_link}}{% elif request.subdir %}/{{request.subdir}}{%endif%}{%else%}{%endif%}/", title:"{{brand.name}}"}`

Reply:
So, should the settings.py be updated to include SUB_DIR in the default value for brand.logo_link? We still need to support subdir without too many configuration hoops because when we get to 4.1 we'll be back under a subdir deployment, but probably won't be overriding logo_link anymore.

Reply:
@andrewkcarter okay changes tested locally and then applied to rapidpro-docker.